tests/network/test_services.py

```diff
@@ -74,13 +80,21 @@ def test_get_transactions_main_failure(self):
         with pytest.raises(ConnectionError):
             MockBackend.get_transactions(MAIN_ADDRESS_USED1)
 
-    def test_get_transactions_test_equal(self):
```

Comment:
I think overwriting `test_get_transactions_test_equal` and `test_get_transactions_test_failure` was a mistake?

Reply:
Oops! Yes, that was a mistake. Good eye! I copied these, made changes and I guess deleted the originals by accident? I've added these back in as-is (the "edited" versions are actually new tests).